src/network/netdev/vxlan.c

```diff
@@ -165,16 +187,28 @@ int config_parse_vxlan_group_address(const char *unit,
 
         r = in_addr_from_string_auto(rvalue, &f, &buffer);
         if (r < 0) {
-                log_syntax(unit, LOG_ERR, filename, line, r, "vxlan multicast group address is invalid, ignoring assignment: %s", rvalue);
+                log_syntax(unit, LOG_ERR, filename, line, r, "vxlan multicast '%s' address is invalid, ignoring assignment: %s", lvalue, rvalue);
                 return 0;
         }
 
-        if (v->family != AF_UNSPEC && v->family != f) {
-                log_syntax(unit, LOG_ERR, filename, line, 0, "vxlan multicast group incompatible, ignoring assignment: %s", rvalue);
-                return 0;
+        r = in_addr_is_multicast(f, &buffer);
+
+        if (STR_IN_SET(lvalue, "Group", "Remote")) {
+                if (r < 0) {
+                        log_syntax(unit, LOG_ERR, filename, line, 0, "vxlan invalid multicast '%s' address, ignoring assignment: %s", lvalue, rvalue);
+                        return 0;
+                }
+
+                v->remote_family = f;
+        } else {
+                if (r > 0) {
+                        log_syntax(unit, LOG_ERR, filename, line, 0, "vxlan invalid '%s' address, ignoring assignment: %s", lvalue, rvalue);
```

Comment:
so, you are saying the local address cannot be multicast? if so, that's whatthe msg should say